.github/workflows/build_and_test.yml

```diff
@@ -882,6 +857,30 @@ jobs:
                                                369495373322.dkr.ecr.eu-central-1.amazonaws.com/${image}:${{ needs.tag.outputs.build-tag }}
           done
 
+  push-to-acr-dev:
```

Comment:
Probably we should trigger `deploy` job after `push-to-acr-dev`/`push-to-acr-prod`

Reply:
It's listing `promote-images` as needs, can add these two there as well?

https://github.com/neondatabase/neon/blob/93da9c0988260387c2b65035897781955b444f16/.github/workflows/build_and_test.yml#L959

Reply:
Yep, I think so. Otherwise, we could start `deploy` including Azure without having images in ACR (if `push-to-acr-*` job fails for some reason).

Btw, for `deploy` job, to make it run with skipped, we probably also need to amend `if` statement a bit and to allow it to be triggered even if one of dependencies got skipped:
```yaml
if: (github.ref_name == 'main' || github.ref_name == 'release'|| github.ref_name == 'release-proxy') && !failure() && !cancelled()
```

Reply:
> Btw, for `deploy` job, to make it run with skipped, we probably also need to amend `if` statement a bit and to allow it to be triggered even if one of dependencies got skipped:
> 
> ```yaml
> if: (github.ref_name == 'main' || github.ref_name == 'release'|| github.ref_name == 'release-proxy') && !failure() && !cancelled()
> ```

I would have assumed that would work magically, thanks!